app/adapters/organisations_adapter.rb

```diff
@@ -0,0 +1,11 @@
+class OrganisationsAdapter
+  def find(slug)
+    response = OrganisationFetcher.fetch(slug)
+    Organisation.new(
+      title: response["title"],
```

Comment:
I believe this is an issue across the codebase. So far I've tended to lean towards keeping the double-quotes to make it easier to see more significant changes. In the same way, I've been using double-quotes across the board, e.g. in spec names.

If we are going to address this issue, I think I'd prefer to do it across the board and ideally enable some kind of Rubocop rule to keep the code consistent. So I hope it's OK, but I'm not going to make the change you suggest in this PR.